api/package.json

```diff
@@ -9,7 +9,7 @@
     "lint": "eslint --fix src -c .eslintrc.json --ext js",
     "format": "prettier --write \"./**/*.{js,jsx,json,md}\"",
     "format:check": "prettier --check \"./**/*.{js,jsx,json,md}\"",
-    "test": "jest"
+    "test": "jest --passWithNoTests"
```

Comment:
which test was failing? i think I fixed it in my mongo connection PR

Reply:
Ok, I'll get your changes in

Reply:
Could you remove the --passWithNoTests flag?